### PR TITLE
Adds Import and Export Roles for Nova Flavors

### PR DESCRIPTION
**Pull request by rbrady**
This patch adds import and export roles, supporting files and tests
for migrating Nova Flavors between clouds.

### CODE REVIEW

**Comment by ccamacho** [comment]
Hi Ryan! This is awesome! Pretty close I think. I remember having this issue and what I did is to install devstack with the same options locally in another dev box, and used the Nova config files in the job. It passed that part at least on my tests back then.

**Comment by jistr** [comment]
What about trying to re-enable n-cond now? It should create the cell1.conf:

https://github.com/openstack/devstack/blob/40879683cc9719b5791ce6755f747958c9ef9cd3/lib/nova#L520-L524

https://github.com/openstack/devstack/blob/40879683cc9719b5791ce6755f747958c9ef9cd3/lib/nova#L733-L736

**Comment by ccamacho** [comment]
Yeah, what I didn't have before was the dummy rabbit service. I think this time is closer than what I managed to move it before.

**Comment by jistr** [comment]
Looking again at the `lib/nova` in devstack, do we need to add "placement" or "placement-client" or both into the enabled services list? I'm not sure where the endpoint is created, but there is some extra config going on here https://github.com/openstack/devstack/blob/master/lib/nova#L540

File: .github/workflows/consistency-functional.yml [hunk 1]
```diff
@@ -168,22 +104,40 @@ jobs:
           cd devstack
           cat << EOF > local.conf
```
Comment (SeanMooney) [inline .github/workflows/consistency-functional.yml:105]: EOF should be 'EOF' to prevent the varibales being substituted

**Pull request merged**